package.json

```diff
@@ -76,7 +76,7 @@
     "@rollup/plugin-image": "^2.1.0",
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^13.0.4",
-    "@schematics/angular": "~14.1.0",
+    "@schematics/angular": "~14.2.0",
     "@storybook/addon-essentials": "~6.5.9",
     "@storybook/addon-knobs": "~6.3.0",
```

Comment:
`ng-packagr` version needs to be updated.

Reply:
Good spot